ProjFS.Mac/PrjFSKext/KauthHandler.cpp

```diff
@@ -756,6 +898,18 @@ KEXT_STATIC int HandleFileOpOperation(
             goto CleanupAndReturn;
         }
     }
+    else if (KAUTH_FILEOP_WILL_RENAME == action)
+    {
+        currentVnode = reinterpret_cast<vnode_t>(arg0);
+        if (VnodeIsEligibleForEventHandling(currentVnode))
+        {
+            const char* fromPath = reinterpret_cast<const char*>(arg1);
+            const char* toPath = reinterpret_cast<const char*>(arg2);
+            KextLog_Info("HandleFileOpOperation: KAUTH_FILEOP_WILL_RENAME vnode %p '%s' -> '%s'", currentVnode, fromPath, toPath);
+
+            RecordPendingRenameOperation(currentVnode);
```

Comment:
Maybe a comment here, 

// We store rename operations, so we can identify if the following PreDelete is part of a rename

Reply:
I've riffed on the wording, [but we now have a comment in this location](https://github.com/microsoft/VFSForGit/pull/1193/commits/753ff3236def50cc95bbfec98480b1f50f2c35d5#diff-ab3be22ed1fbe1779471dbe568982eb0R962).